aten/src/ATen/core/ivalue.h

```diff
@@ -1359,6 +1359,7 @@ struct TORCH_API IValue final {
     Payload(Payload&&) = delete;
     Payload& operator=(const Payload&) = delete;
     Payload& operator=(Payload&&) = delete;
+    // NOLINTNEXTLINE(modernize-use-equals-default)
```

Comment:
Why not fix this one? Does it trigger bugs?

Reply:
Oh because it's a union, not a struct. Yeah, I think clang-tidy has a fix for this false positive in newer versions.